src/compiler/parser.ts

```diff
@@ -9107,6 +9107,9 @@ namespace Parser {
                     case "satisfies":
                         tag = parseSatisfiesTag(start, tagName, margin, indentText);
                         break;
+                    case "internal":
+                        tag = parseSimpleTag(start, factory.createJSDocInternalTag, tagName, margin, indentText);
+                        break;

```

Comment:
I was initially concerned that this didn't match any of the other tags, but this is actually more like `deprecated` above which does take this form.